docker/images/Dockerfile.backend

```diff
@@ -6,37 +6,16 @@ ENV LANG='C.UTF-8' \
     JAVA_HOME="/opt/jdk" \
     DEBIAN_FRONTEND=noninteractive \
     NODE_VERSION=v22.16.0 \
-    IMAGEMAGICK_VERSION=7.1.1-47 \
     TZ=Etc/UTC
 
 RUN set -ex; \
     apt-get -qq update; \
     apt-get -qq upgrade; \
     apt-get -qqy --no-install-recommends install \
-        curl \
-        ca-certificates \
-        binutils \
-        build-essential autoconf libtool pkg-config \
-        libltdl-dev \
-        libpng-dev libjpeg-dev libtiff-dev libwebp-dev libopenexr-dev libfftw3-dev \
-        libzip-dev \
-        liblcms2-dev liblzma-dev libzstd-dev \
-        libheif-dev librsvg2-dev \
+        curl ca-certificates binutils \
```

Comment:
style suggestion: if possible, these deps should be in different lines and in alphabetical order.